Initial setup

PaulTerrasi opened the pull request:
@michaelsilver I tried to imitate everything that you did in setting up the dodgeball repo. Everything seems to be working, but obviously you should still look it all over.

Closes #2

michaelsilver commented on README.md:
```diff
@@ -1,2 +1,62 @@
-# Overclocked-Swerve
+# OverclockedSwerve
+[![Build Status](https://travis-ci.org/246overclocked/OverclockedSwerve.svg)](https://travis-ci.org/246overclocked/OverclockedSwerve)
```
Should be
```
[![Build Status](https://travis-ci.org/246overclocked/OverclockedSwerve.svg?branch=master)](https://travis-ci.org/246overclocked/OverclockedSwerve)
```

michaelsilver commented:
Really great job with this!

You should copy over the `.gitignore` from dodgeball, which has all you need -- plus, it will allow you to commit the essential `.classpath` and `.project`. Otherwise, people can't open the project in Eclipse.

Finally, I have a minor correction in the README.

If you do those things, then looks good to me! :rocket:

michaelsilver commented:
P.S. I didn't fully verify, but I assume you just copied the `lib` folder from dodgeball? If so, you're all good!

PaulTerrasi commented:
Yes, I did copy it directly. And I fixed the Travis badge.

For `.gitignore`, I removed `.classpath` and `.project`. However, both of the launch configuration files and multiple files in `lib` were supposed to be ignored. To keep the `.gitignore` file consistent with what is actually in the project, I made exceptions for all of those files. If you are OK with that, then I will merge.

michaelsilver commented:
Great thanks for the fixes! Looking it over again, the `.gitignore` makes sense (I made one small comment), but you still haven't added the `.project` and `.classpath` files to the repo, so I still can't open it in Eclipse.

michaelsilver commented:
Once you add the `.project` and `.classpath` files to the repo, and address my minor comment about the .gitignore, you should be all set to merge.

PaulTerrasi commented:
I removed that thing from .gitignore because it also wasn't in Dodgeball's
.gitignore file. I am just going to leave it out, I'm pretty certain it's
not important.

I will add those files later tonight.
On Jan 5, 2016 4:38 PM, "Michael Silver" <notifications@github.com> wrote:

> Once you add the .project and .classpath files to the repo, and address
> my minor comment about the .gitignore, you should be all set to merge.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/246overclocked/OverclockedSwerve/pull/3#issuecomment-169141273>
> .
>
